docs/releasehistory.md

```diff
@@ -9,10 +9,12 @@ Releases follow the `major.minor.micro` scheme recommended by [PEP440](https://w
 
 ## Current development
 
+### Improved documentation and warnings
+- [PR #1442](https://github.com/openforcefield/openff-toolkit/pull/1442): Doctests added to CI, leading to numerous fixed docstrings and examples.
```

Comment:
```suggestion
- [PR #1442](https://github.com/openforcefield/openff-toolkit/pull/1442): Doctests added to CI, leading to numerous fixed docstrings and examples therein.
```

Non-blocking: I interpreted this to mean the notebooks were updated, which was a little alarming since I thought they were in good shape (they are).